cmake/AddInputFileTests.cmake

```diff
@@ -1,6 +1,10 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
+option(SPECTRE_INPUT_FILE_TEST_TIMEOUT_FACTOR
+  "Multiply timeout for input file tests by this factor"
+  1)
```

Comment:
Set the default to `OFF` and then below you can probably check
```
if (NOT ${SPECTRE_INPUT_FILE_TEST_TIMEOUT_FACTOR})
```
As far as I've been able to tell, these `option` things are only ever `ON` or `OFF` (much to my annoyance)

Reply:
As far as I understood, cmake's `if` interprets any non-zero number as `OFF` and a one as `ON`: https://cmake.org/cmake/help/latest/command/if.html
When setting the variable e.g. as `-D SPECTRE_INPUT_FILE_TEST_TIMEOUT_FACTOR=2` the "2" is correctly set as a string though. Only a "1" would be set as "ON". That's why I check for "ON", and use the number in the math expression otherwise. Seems to work.